### PR TITLE
fix(functions): email notification - formatting issues (BOAS-1720)

### PR DESCRIPTION
**Pull request by BenSurgisonPINS**
## Describe your changes

- Backout decoding
- Tested via unit tests
- Will tests sending email notification once deployed to DEV as it's the only way to test properly

## BOAS-1720 Project updates: email notification - issue with hyphens, apostrophes and brackets

https://pins-ds.atlassian.net/browse/BOAS-1720

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js [hunk 1]
```diff
@@ -75,12 +75,11 @@ export class NotifySubscribers {
 		if (!this.messageIsValid()) {
 			return;
 		}
-		const update = await this.getExtendedUpdate();
```
Comment (emil-placheta) [inline apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js:78]: the extended update is required to get the project name to be passed to the rest of the update data. Why was that undone?

File: apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js [hunk 2]
```diff
@@ -180,14 +179,14 @@ export class NotifySubscribers {
...
 				personalisation: {
-					projectName,
+					// todo: project name & link ?
+					projectName: caseReference,
 					projectLink,
-					title: `${projectName} - Project Update`,
-					subject: `${projectName} - Project Update Notification`,
+					// todo: what should these be?
+					title: `${caseReference} - Project Update`,
+					subject: `${caseReference} - Project Update Notification`,
```
Comment (emil-placheta) [inline apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js:189]: The email needs to have the project name displayed as the title and subject, not the case reference, unless that has been reverted?

File: apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js [hunk 3]
```diff
@@ -75,12 +75,11 @@ export class NotifySubscribers {
...
 		}
-		const update = await this.getExtendedUpdate();
+		const update = await this.getUpdate();
 
 		if (update === null) {
-			throw new Error(
-				`update with id '${this.msg.id}' not found on case ${this.msg.caseReference}`
-			);
+			this.logger.error(`update (id ${this.msg.id}) doesn't exist with`);
+			return;
```
Comment (emil-placheta) [inline apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js:82]: reverted error message?

**Review by emil-placheta — COMMENTED** [review]
sorry If I missed something, but it looks like you have reverted 1 commit too far and undone the changes I have added here last, that include the project name in the template data?

File: apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js (hunk 1 above)
Comment (BenSurgisonPINS) [reply apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js]: My fault, that should have stayed in.  I have restored those changes.

File: apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js (hunk 2 above)
Comment (BenSurgisonPINS) [reply apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js]: As above, it shouldn't have been reverted.  Now restored.

**Review by emil-placheta — APPROVED** [review]
lgtm!

**Pull request merged**